### ISSUE TITLE
While training accuracy increases (training loss decreases), validation accuracy remains more or less constant (validation loss increases)

### ISSUE THREAD

**Issue by PareshKamble** [keras-team/keras-preprocessing#230]
Hi!
I am trying to predict imdb rating from movie posters using a CNN. I know it may not be a good prediction problem. People have done this prediction using movie [metadata](https://nycdatascience.com/blog/student-works/web-scraping/movie-rating-prediction/).Though, I wanted to give it a try. 

I downloaded movie posters and its corresponding imdb rating of 5039 movies from files provided in [this](https://github.com/sundeepblue/movie_rating_prediction) repository. The imdb ratings were floored to obtain 10 classes. Thus, I treated this as a multi(10)-class classification problem. The train-validation-test images are divided into 3800-1000-239 respectively. 

I was expecting accuracy of at least 50%. However, my validation accuracy is struck at around 31% where as training accuracy reaches around 97% (may be over-fitting). However, validation loss keeps on increasing. Test set accuracy is around 31-34%, whereas the test predictions are mostly +/- 1 rating correct for around 70% of time.

It look like a high variance issue. I tried increasing the training data (from 3300 to 3800 currently), used regularizers, applied dropout (0.25-0.5) but no luck so far. I even used pretrained networks like VGG16 and VGG19 but could not improve the validation accuracy (decrease the validation loss).

The accuracy and loss plots are shown below.
![image](https://user-images.githubusercontent.com/26019729/62423498-b4926a00-b6de-11e9-8c7f-8465d2a808bb.png)
![image](https://user-images.githubusercontent.com/26019729/62423502-c1af5900-b6de-11e9-901f-8f3bceaf782a.png)

Kindly, provide insights into this issue.

You may find my_repository [here](https://github.com/PareshKamble/Predicting_imdb_rating_from_movie_posters).

Thank you!

- [x] Check that you are up-to-date with the master branch of keras-preprocessing. You can update with:
`pip install git+git://github.com/keras-team/keras-preprocessing.git --upgrade --no-deps`

- [x] Provide a link to a GitHub Gist of a Python script that can reproduce your issue (or just copy the script here if it is short).

**Comment by rragundez** [comment]
this is not an issue with the source code, therefore I will close it. People can still comment on it though.
